kfs-cam/src/main/java/org/kuali/kfs/module/cam/businessobject/AssetBase.java

```diff
@@ -151,7 +152,9 @@
     protected ObjectSubType financialObjectSubType;
     protected AssetAcquisitionType acquisitionType;
     protected ContractsAndGrantsAgency agency;
+    protected List<Note> boNotes = new ArrayList<Note>();

```

Comment:
Doing a search in the entirety of KFS, the only place that buildListOfDeletionAwareLists() is used can be found in various *Document.java, and when done so, it appears that it is only for maintenance of Lists on sub-objects. In this case, boNotes is a list directly on this object, and as such doesn't require special handling using this method, Spring handles it.